### PR TITLE
pycharm ide + seperation of ide pages

### PR DESCRIPTION
**Pull request by glicht**
## Status
Ready

## Related Issues
related: https://github.com/demisto/etc/issues/23431

## Description
Add the PyCharm Article. Separate the IDE pages.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: docs/integrations/pycharm-plugin.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+1.  Open the project you want to work on in PyCharm. For example, `Cortex XSOAR Content` repository.
+2.  In PyCharm, click **Preferences > Tools > Cortex XSOAR Plugin Setup**.
+3.  (MacOS users) When prompted for access to your keychain, select **Allow Always**. This will avoid issues later on.
+4.  Enter your Cortex XSOAR server URL and port, if necessary.  
+    If you are working on a dev instance on localhost, use HTTP (not HTTPS), for example: ([http://localhost:8080](http://localhost:8080/)).
+5.  Enter your Cortex XSOAR API key.
+6.  Optional: To test the configuration, click **Test**.
+7.  Click **Apply**.
+
+After you successfully configure the plugin several files are automatically downloaded from your Cortex XSOAR instance and saved in your project's root. These files enable you to use the functions you defined in Cortex XSOAR and run the scripts locally in PyCharm.
```
Comment (amshamah419) [inline docs/integrations/pycharm-plugin.md:59]:
```suggestion
After you successfully configure the plugin, several files are automatically downloaded from your Cortex XSOAR instance and saved in your project's root directory. These files enable you to use the functions you defined in Cortex XSOAR and run the scripts locally in PyCharm.
```

File: docs/integrations/pycharm-plugin.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+New Python and YML files are created. By default, the editor opens the new Python file. You're ready to start writing code.
+
+## Python Files
+
+You can create new Python files, or open existing Python files.
+
+### Create a Python file
+
+1.  Under the **Cortex XSOAR Settings** toolbar, you’ll have two button named `Create Cortex XSOAR Automation Configuration` and `Create Cortex XSOAR Integration Configuration`
```
Comment (amshamah419) [inline docs/integrations/pycharm-plugin.md:83]:
```suggestion
1.  Under the **Cortex XSOAR Settings** toolbar, you’ll have two buttons named `Create Cortex XSOAR Automation Configuration` and `Create Cortex XSOAR Integration Configuration`
```

File: docs/integrations/pycharm-plugin.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+2.  Enter the necessary arguments.
+3.  Click **Export and run in Cortex XSOAR**.
+
+![PyCharm_-_Export_and_Run.png](../doc_imgs/integrations/pycharm-export.png)
+
+## Update Cortex XSOAR Mock Files
+
+In general, you do not need to update Cortex XSOAR mock files. When there is an important update to the mock files, we will announce it in the plugin release notes.
+
+You might want manually update the mock files after content updates, in which `CommonPythonServer` and `CommonServerUserPython` files are changed or updated. Manually updating the mock files updates the files in your local environment.
```
Comment (amshamah419) [inline docs/integrations/pycharm-plugin.md:117]:
```suggestion
You might want to manually update the mock files after content updates, in which `CommonPythonServer` and `CommonServerUserPython` files are changed or updated. Manually updating the mock files updates the files in your local environment.
```

File: docs/integrations/pycharm-plugin.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+Instead of running commands in Cortex XSOAR, you can run them locally in PyCharm. Every demisto action `(demisto.___)` is executed by mock files, and the code runs as a regular Python script.
+
+Since the script is run as a regular Python file, you can debug the code by adding the relevant parameters in the \`demistomock.py\` file, which is located in your project's root. In most cases, you need to edit the return value for several functions.
+
+|Function|Description|Example|
+|--- |--- |--- |
+|command|Called for `demisto.command()`|return `"my_command_name"`|
+|args|Called for `demisto.args()`|return `{ "myArg": "example"}`|
+|parameters|Called fort `demisto.params()`|return `{ "api_key": "example"}`|
```
Comment (amshamah419) [inline docs/integrations/pycharm-plugin.md:133]:
```suggestion
|parameters|Called for `demisto.params()`|return `{ "api_key": "example"}`|
```

File: docs/integrations/xsoar-ide.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+We recommend to try out the [PyCharm IDE Plugin](pycharm-plugin). It simplifies third-party integration and script development by enabling users to author Python content for Cortex XSOAR directly in PyCharm. Highly recommended for developing [Python Unit Tests](unit-testing). 
+:::
+
+
+It is important to take a moment and familiarize yourself with the Cortex XSOAR IDE as it may work differently than other IDEs you are familiar with. Some key differences between the Cortex XSOAR IDE and, for example PyCharm, is the absence of an interpreter.
+
+## The Script Helper
+Cortex XSOAR is equipped with a script helper which is accessible via the button below:
+
+![](../doc_imgs/integrations/script-helper.png)
```
Comment (amshamah419) [inline docs/integrations/xsoar-ide.md:17]: Do we want image alt tags for SEO purposes, or are they out of scope?

File: docs/integrations/pycharm-plugin.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+    If you are working on a dev instance on localhost, use HTTP (not HTTPS), for example: ([http://localhost:8080](http://localhost:8080/)).
+5.  Enter your Cortex XSOAR API key.
+6.  Optional: To test the configuration, click **Test**.
+7.  Click **Apply**.
+
+After you successfully configure the plugin several files are automatically downloaded from your Cortex XSOAR instance and saved in your project's root. These files enable you to use the functions you defined in Cortex XSOAR and run the scripts locally in PyCharm.
+
+*   `CommonServerPython`
+*   `CommonServerUserPython`
+*   `Cortex XSOARmock` (enables you to run the scripts locally in PyCharm)
```
Comment (anara123) [inline docs/integrations/pycharm-plugin.md:63]:
```suggestion
*   `demistomock` (enables you to run the scripts locally in PyCharm)
```

File: docs/integrations/xsoar-ide.md (hunk 5 above)
Comment (glicht) [reply docs/integrations/xsoar-ide.md:17]: not that important

**Pull request merged**